src/content/self-hosted/manage/crds.mdx

```diff
@@ -128,3 +132,55 @@ spec:
 ```
 
 To apply the changes manually, run the command `kubectl apply -f my-catalog-item.yaml` from your terminal to add the catalog item to the your Catalog in the cluster.
+
+
+## Cloud Credentials
+
+Cloud Credentials are mainly managed via the [Admin dashboard](admin/dashboard.mdx#cloud-credentials), but can also be managed via one of the CRDs installed by Okteto: `dynamicconfigs.admin.okteto.com`.
+Using CRDs can be useful if you would like to add Cloud Credentials manually with a simple `kubectl` command or manage your Cloud Credentials using GitOps.
+
+:::note
+If you add credentials using CRDs they will be displayed in the UI, but they can't be modified from the UI. If you want to manage them from the UI, they must be created from there.
+:::
+
+For example, if you want to add a configuration for GCP Cloud Credential, you will need to create a resource like this:
+
+- `metadata.name`: `gcp-credentials-config`
```

Comment:
I think there are 2 important things to remark related to Cloud credentials:
* In order to work, the name for each one has to be a specific one (`gcp-credentials-config` and `aws-credentials-config`)
* You can only specify one credentials per provider. It means 1 for GCP and 1 for AWS maximum. You can have one for each of them, but you cannot have more than 1 for each type

Reply:
Thanks. Added a note to clarify this